### ISSUE TITLE
allow to override BSConfig::localarch

### ISSUE THREAD

**Issue by k0da** [openSUSE/open-build-service#1684]
In case we've found fat machine with architecture different than the one set it BSConfig::localarch, it would be nice to have an ability to build products there. By overriding localarch through project config or something like that

**Comment by k0da** [comment]
As bonus it will unblock ports from waiting on main (openSUSE:Factory) tree to finish a kiwi build

**Comment by k0da** [comment]
some WIP
https://github.com/k0da/open-build-service/commit/51cbc57daa4f5bde46b3a11ce9aee44cd6c43c4f
